scipy/stats/_entropy.py

```diff
@@ -86,18 +89,22 @@ def differential_entropy(
     window_length: Optional[int] = None,
     base: Optional[float] = None,
     axis: int = 0,
+    method: str = "vasicek",
 ) -> Union[np.number, np.ndarray]:
     r"""Given a sample of a distribution, calculate the differential entropy.
 
-    This routine uses the Vasicek estimator of the differential entropy. Given
-    a sorted random sample :math:`X_1, \ldots X_n` this is defined as:
+    By default, this routine uses the Vasicek estimator of the differential
+    entropy. Given a sorted random sample :math:`X_1, \ldots X_n`, this is
+    defined as:
```

Comment:
I would actually remove it entirely if we change the default. I don't think we need to define differential entropy here, do you?

Reply:
Either fixing or removing is fine for me. In a lot of functions we don't have maths. I am not sure if we want to move away from the maths in general or not though.

Reply:
I would love to include the math, formatted as math. However, [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) disagrees, suggesting that LaTeX should only be used in the Notes section, and even then:
![image](https://user-images.githubusercontent.com/6570539/114494370-1c49c980-9bd1-11eb-9efc-8be3836de8e8.png)
Rather than risking additional rebuke for my use of LaTeX, I would prefer to remove it entirely if it's not needed.

Reply:
You have more experience with `scipy.stats`, I will let you judge and remove the maths if not needed :)

Just to be sure, I am talking about the maths in general so this and the rest with the main equation (`\frac{1}{n}\sum_1^n ...`)

Reply:
@tupui [This](https://github.com/scipy/scipy/pull/13298#pullrequestreview-638207522) is a perfect example of why I now avoid including math unless necessary. Please briefly note the amount of work that has gone into that PR. In that case, the comment is about the notes section, where LaTeX is allowed, and it is a statistical distribution, where LaTeX is always used to define the distribution. So I would prefer to just remove the LaTeX here.